### PR TITLE
Add option to search in likes, also fixes duplicated tracks

### PR DESCRIPTION
**Pull request by Pitros**
Fix for #564 and adds feature #509.

Whoops :D

### CODE REVIEW

**Comment by weblancaster** [comment]
Can you fix the merge conflicts please @Pitros

**Comment by Pitros** [comment]
@weblancaster yeah. Something weird just happend with my repo i think, need to fix that.

**Comment by Pitros** [comment]
@weblancaster now it's fixed.

File: app/public/stylesheets/sass/_components/_view-container.scss [hunk 1]
```diff
@@ -12,11 +12,53 @@
...
   }
 
-  & > div > h1 {
+  & > div > h1, & > div > header > h1 {
     margin: 10px 0 30px;
     font-weight: 300;
     font-size: 3em;
   }
+
+    & > div > header {
```
Comment (weblancaster) [inline app/public/stylesheets/sass/_components/_view-container.scss]: selectors not following name convention e.g `favoritesView_componentName` https://github.com/Soundnode/soundnode-app/wiki/HTML-CSS-convention

**Comment by Pitros** [comment]
@weblancaster, better?

**Comment by weblancaster** [comment]
Hey @Pitros well you still have a lot of specificity and hierarchy.

You should do this
```css
.mainView {
    // flex: 5 0 0;
    padding: 10px 20px;
    margin-left: 200px;
    overflow-y: scroll;
    overflow-x: hidden;
    height: 100%;

    & > div {
        min-width: 130px;
        max-width: 100%;
        margin: 0 auto;
    }

    & > div > h1 {
        margin: 10px 0 30px;
        font-weight: 300;
        font-size: 3em;
    }

    .filters {
        display: flex;

        h1 {
            margin: 10px 0 30px;
            font-weight: 300;
            font-size: 3em;
        }
    }

    .filters_inner {
        line-height: 65px;
        text-align: right;
        width: 100%;
    }

    .filters_header {
        font-weight: 300;
        font-size: 11px;
        letter-spacing: 1px;
        text-transform: uppercase;
        margin: 0 10px 0 0;
        display: inline-block;
    }
    .filters_textFilter {
        font-weight: 300;
        color: #FFF;
        margin: 0 10px 4px 0;
        display: inline-block;
        & input {
            display: block;
            background: transparent;
            border: none;
            border-bottom: 1px solid #3e3e40;
            color: #FFF;
            border-radius: 0;
            padding: 0;
            margin: 0;
            cursor: text;
            &.active {
                border-color: #F50;
            }
            &:focus {
                outline: none;
                border-color: #a9aaae;
                box-shadow: none;
            }
        }
    }
}
```

and I still seeing duplicates.. below is the screenshot.

screen shot 2016-02-19 at 10 58 51 am

**Comment by Pitros** [comment]
You see those duplicates in search results, right? But we're talking about liked tracks - after scrolling down each track was displayed two times (code bug).

**Comment by weblancaster** [comment]
@Pitros oh got it.. this duplicated bug is when on user page

**Comment by Pitros** [comment]
@weblancaster user page has the same code bug (which was added with partial filtering) so I can remove those lines (fix it) and make commit it to this PR if you want.

**Comment by weblancaster** [comment]
Would be ideal on a new PR @Pitros

**Comment by Pitros** [comment]
Ok then. You can check this PR again.

**Comment by SaahilMehta** [comment]
Am getting duplicates on liked tracks. Only seems to appear on the pages following the first, preloaded page when you first hit the likes tab.

**Comment by weblancaster** [comment]
@Pitros the issue still happening.. can you take a look please otherwise I can take this issue over. please let me know.

**Pull request closed without merging**